md/bdm-management-in-bonita-bpm-portal.md

```diff
@@ -5,6 +5,10 @@ Only one model can be deployed at a time in the Portal, so make sure it contains
 
 The Business objects must match the structure used by the deployed processes. Make sure that when modifying the Business Model, the process is modified accordingly.
 
+::: warning
+**:fa-exclamation-triangle: Warning:**  The BDM update in a production environment is not supported, and can lead to unexpected behaviour. It should generally speaking be avoided when possible. If the need arises please contact the bonitasoft Support team to help you with the process.
```

Comment:
This chapter should be added after the chapter of installation is over: Let's explain the happy path of installing and then warn for updating.